server/src/common/model/schema/appointments/appointment.schema.ts

```diff
@@ -212,6 +212,11 @@ export const appointmentSchema = new Schema<IAppointments>({
     default: null,
     description: "Adresse email CFA",
   },
+  is_anonymized: {
```

Comment:
Isn't it better to use: "anonymized_at": Date instead of a boolean? :)

Reply:
Business doesn't care when it has been anonymized, but with the timestamp global update we'll have the info later on 👍